content/en/docs/components/katib/user-guides/metrics-collector.md

```diff
@@ -6,16 +6,23 @@ weight = 40
 
 This guide describes how Katib metrics collector works.
 
-## Metrics Collector
+## Overview
+
+There are two ways to collect metrics:
+
+1. Pull-based: collects the metrics using a _sidecar_ container. A sidecar is a utility container that supports
+the main container in the Kubernetes Pod.
+
+2. Push-based: users push the metrics directly to Katib DB in the training scripts.
 
 In the `metricsCollectorSpec` section of the Experiment YAML configuration file, you can
 define how Katib should collect the metrics from each Trial, such as the accuracy and loss metrics.
 
-Your training code can record the metrics into `StdOut` or into arbitrary output files. Katib
-collects the metrics using a _sidecar_ container. A sidecar is a utility container that supports
-the main container in the Kubernetes Pod.
+## Pull-based Metrics Collector
+
+Your training code can record the metrics into `StdOut` or into arbitrary output files. 
 
-To define the metrics collector for your Experiment:
+To define the pull-based metrics collector for your Experiment:
 
 1. Specify the collector type in the `.collector.kind` field.
    Katib's metrics collector supports the following collector types:
```

Comment:
Please can you fix the formatting for JSON example, it looks like it renders incorrectly.
```
{"epoch": 0, "foo": "bar", "fizz": "buzz", "timestamp": "2021-12-02T14:27:51"}
{"epoch": 1, "foo": "bar", "fizz": "buzz", "timestamp": "2021-12-02T14:27:52"}
{"epoch": 2, "foo": "bar", "fizz": "buzz", "timestamp": "2021-12-02T14:27:53"}
{"epoch": 3, "foo": "bar", "fizz": "buzz", "timestamp": "2021-12-02T14:27:54"}
```

Reply:
Okay for that.